### DISCUSSION TITLE
Welcome to seams-core Discussions!

### DISCUSSION THREAD

**Discussion by HaoZeke** [d-SEAMS/seams-core#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MengqiangH** [comment]
Consult about some functions (chill+, in-plane rdf) in d-seams.

Dear developers,

Thanks for your efforts in developing the software. I am a Ph.D. student, and my research interest is the molecular simulation of ice nucleation. After reading your paper (J. Chem. Inf. Model. 2020, 60, 2169−2177) and installing d-seams, I am still a little confused, and hope you can help me.

In the vars.lua in the example “ChillPlus”, there are several outputs, including chillPlis_mod, chillPlus_noMod, chill_noMod and so on. In my opinion, the chill_noMod and chillPlus_noMod outputs should be the results of chill and chill+ algorithm, respectively. But I don’t know the information about chillPlus_mod, what’s the major improvements or modifications in the modified chill+ algorithm? Maybe I have missed some important references, hope you can kindly point out. Besides, the outputted trajectory files don’t include chill classification, could I get this by setting some other variables?

The other consulting is about the calculation about in-plane radial distribution function. Is this function only available for two-dimensional case? Because I noticed that in some paper, the authors have calculated the in-plane RDF for a three-dimensional case. Can d-seams execute similar calculation?

Thanks for your attention . I am looking forward to your reply.

Best regards,
Mario
